### PR TITLE
5187: Added check for location on events in sections

### PR DESCRIPTION
**Pull request by arniloftsson**
#### Link to issue

https://platform.dandigbib.org/issues/5187

#### Description

This issue is solved. Now event location and event place is displayed. Because off problems with legacy dependencies in the sass code off the module the css fixes are place in the theme in sass/addons/nodelist.scss.

#### Screenshot of the result

image

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ding_nodelist/ding_nodelist.module [hunk 1]
```diff
@@ -1488,6 +1488,48 @@ function _ding_nodelist_format_event_time(array $date) {
...
 
+/**
+ * Gets the location off a event if defined else the library.
+ *
+ * @param mixed $item
+ *   The event item.
+ * @param bool $islink
+ *   Return formatted as link.
+ *
+ * @return string
```
Comment (JrgenGNielsen) [inline modules/ding_nodelist/ding_nodelist.module:1499]: return render array

**Review by JrgenGNielsen — CHANGES_REQUESTED** [review]
There's a couple of Scrutinizer errors. You could perhaps rebase to just one commit.
Also: Fix _ding_nodelist_get_event_location return value

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ding_nodelist/ding_nodelist.module (hunk 1 above)
Comment (arniloftsson) [reply modules/ding_nodelist/ding_nodelist.module]: There is Scrutinizer error message. But is says that Drupal code guidelines are not compatible with the current version.

**Review by JrgenGNielsen — APPROVED** [review]
reviewet og godkendt

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: f7b67c525d7fb0e5cb1aea3fed40ec5e25eb4216
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: bb478b5d23952129fa8af7ec3813054f20cb2ef2
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by arniloftsson** [comment]
Added minor styling changes to fix issues here: https://platform.dandigbib.org/issues/5187#note-17

**Review by JrgenGNielsen — APPROVED** [review]
reviewed and approved

**Pull request merged**